source/utilities.tex

```diff
@@ -4489,7 +4493,7 @@
 \tcode{is_trivially_move_assignable_v<$\tcode{T}_i$> \&\&}
 \tcode{is_trivially_destructible_v<$\tcode{T}_i$>}
 is \tcode{true} for all $i$, this assignment operator is trivial.
-The expression inside \tcode{noexcept} is equivalent to:
+The expression inside \tcode{noexcept} is equivalent to
```

Comment:
The other few inline cases don't use colon.

Reply:
I would expect consistency with https://github.com/cplusplus/draft/wiki/Specification-Style-Guidelines#writing-effects-in-a-function-description

Reply:
You mean that it should be a code block because it's not a plain expression of type `void`?

Reply:
No, it doesn't say you must use a code block for non-void expressions, it says you must use `return expression` not just `expression`, but that part of the guideline is referring to return values anyway.

I mean this part:

>  When code is used to describe an effects element, a colon (:) should follow the introductory phrase if the code is a statement or a code block, whereas expressions should end with a period (.).

i.e. there should be a colon when the expression is in a code block, but not when it's inline. So I agree with removing the colon here.